app/models/item.rb

```diff
@@ -3,8 +3,10 @@ class Item < ApplicationRecord
   has_many :invoice_items
   has_many :invoices, through: :invoice_items
 
+  default_scope -> { order(:id)}
```

Comment:
This was required to get the correct Items first, as many were created and updated at the same time. These will come into play later on the other resources as well.